app/src/main/java/org/android/go/sopt/LoginActivity.kt

```diff
@@ -0,0 +1,124 @@
+package org.android.go.sopt
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.content.SharedPreferences
+import android.os.Bundle
+import android.view.MotionEvent
+import android.view.inputmethod.InputMethodManager
+import android.widget.Toast
+import androidx.activity.result.ActivityResult
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.contract.ActivityResultContracts
+import androidx.appcompat.app.AppCompatActivity
+import com.google.android.material.snackbar.Snackbar
+import org.android.go.sopt.databinding.ActivityLoginBinding
+
+class LoginActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityLoginBinding
+    private lateinit var savedId: String
+    private lateinit var savedPw: String
+    private lateinit var savedName: String
+    private lateinit var savedSpecialty: String
+    private lateinit var startForResult: ActivityResultLauncher<Intent>
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityLoginBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        setActivityResult()
+        clickSignUp()
+        clickLogin()
+        setAutoLogin()
+    }
+
+    override fun dispatchTouchEvent(ev: MotionEvent?): Boolean {
```

Comment:
이렇게 하지 않고, 간단하게 
```suggestion
    binding.root.setOnClickListener{
        hideKeyboard()
    }
```
해도 된답니다~